clang/lib/DPCT/IncMigration/ExternalReplacement.cpp

```diff
@@ -62,6 +64,9 @@ int save2Yaml(
   }
 
   TUR.DpctVersion = clang::dpct::getDpctVersionStr();
+
+  TUR.CudaVersion = "cuda-" + std::to_string(SDKVersionMajor) + "." +
```

Comment:
How about add 2 fields in yaml? Then we can avoid using regex to extract the number.

Reply:
std::pair<std::string, std::string> type is not supported in YAML serialization and will define two vars to avoid extraction operation.